app/components/edit-reminder.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+  store: Ember.inject.service(),
+  router: Ember.inject.service('-routing'),
```

Comment:
Both Eric and I agree that it IS interesting.  We had a heck of a time trying to route back to /reminders/id/.  This one the only (of many different attempted ways) we could get it to work.  We tried with a controller and had no success.  We can certainly take another look, though.